src/test/java/org/folio/entitlement/integration/keycloak/KeycloakServiceTest.java

```diff
@@ -330,11 +345,11 @@ void negative_failedToCreateScopeWithWebApplicationError() {
 
       var currentDescriptor = moduleDescriptor("mod-foo-1.2.0");
       var prevDescriptor = moduleDescriptor("mod-foo-1.1.0");
-      assertThatThrownBy(() -> keycloakService.updateAuthResources(prevDescriptor, currentDescriptor, TENANT_NAME))
-        .isInstanceOf(IntegrationException.class)
-        .hasMessage("Failed to update authorization scopes in Keycloak")
-        .satisfies(error -> assertThat(((IntegrationException) error).getErrors()).containsExactly(
-          parameter("scope", "Failed to create scope GET (status: 502, info: Bad Gateway)")));
+      assertThatThrownBy(
+        () -> keycloakService.updateAuthResources(prevDescriptor, currentDescriptor, TENANT_NAME)).isInstanceOf(
+        IntegrationException.class).hasMessage("Failed to update authorization scopes in Keycloak").satisfies(
+          error -> assertThat(((IntegrationException) error).getErrors()).containsExactly(parameter("scope",
+            "Failed to create scope GET (cause: WebApplicationException, causeMessage: HTTP 502 Bad Gateway)")));
```

Comment:
same as below, please don't change formatting if not necessary

Reply:
I've formatted the whole file with our standard formatter. If this formatting is somehow wrong - the standard formatter config should be fixed.

Reply:
Actually it's a good question why aren't you using the standard formatter config? We have a standard one to avoid these kind of changes, but apparently it's not used.

Reply:
Does not consistent with code uniformity 
https://folio-org.atlassian.net/wiki/spaces/FOLIJET/pages/509149235/Eureka+Code+Uniformity

Example from page
![image](https://github.com/user-attachments/assets/4f126f18-ea4b-4631-9968-ca10263b0e10)